core/src/token_info/cached.rs

```diff
@@ -141,6 +153,32 @@ impl TokenInfoFetching for TokenInfoCache {
     fn all_ids<'a>(&'a self) -> BoxFuture<'a, Result<Vec<TokenId>>> {
         self.inner.all_ids()
     }
+
+    fn find_token_by_symbol<'a>(
+        &'a self,
+        symbol: &'a str,
+    ) -> BoxFuture<'a, Result<Option<(TokenId, TokenBaseInfo)>>> {
+        const CONCURRENT_REQUESTS: usize = 1;
```

Comment:
Any reason for using a different value than [here](https://github.com/gnosis/dex-services/blob/b60f00f8ddf517cab6ae12cd61e46f9e24d024d3/price-estimator/src/main.rs#L122)

Reply:
No, in my head I thought "since we cache all tokens at startup anyway, I don't expect having concurrent requests will be much benefit" and "this way the implementation is similar to the default one".

But now that I think about it, maybe we can use a similar value to the code you linked.

Reply:
They use the same value now.